cmd/token.go

```diff
@@ -24,11 +26,18 @@ func init() {
 
 	loginCmd.Flags().BoolVarP(&isUserToken, "user-token", "u", false, "Whether to login as a user or getting an app access token.")
 	loginCmd.Flags().StringVarP(&userScopes, "scopes", "s", "", "Space seperated list of scopes to request with your user token.")
+	loginCmd.Flags().StringVarP(&revokeToken, "revoke", "r", "", "Instead of generating a new token, revoke the one passed to this parameter.")
+	loginCmd.Flags().StringVarP(&overrideClientId, "client-id", "i", "", "Override/manually set client ID for token actions. By default client ID from CLI config will be used.")
```

Comment:
Here I'd prefer to leave as just --client-id and no shorthand- "i" to me doesn't feel appropriate as it's not clear what that is for.

Reply:
Yes, it's to be able to quickly revoke any token from any client-ID you might be working with, instead of just the one currently configured for the CLI.
I actually had this use case myself, last week, when my token was visible on-stream.

I will remove the shorthand.